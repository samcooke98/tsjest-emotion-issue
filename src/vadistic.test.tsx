import * as React from "react";

import { create } from "react-test-renderer";
import { createSerializer } from "jest-emotion";
import * as emotion from "emotion";

expect.addSnapshotSerializer(createSerializer(emotion));

import styled from "react-emotion";

it("shorthand div", () => {
  const Box = styled.div({
    padding: "20px",
    backgroundColor: "silver"
  });

  const tree = create(<Box> Box </Box>).toJSON();

  expect(tree).toMatchSnapshot();
});
