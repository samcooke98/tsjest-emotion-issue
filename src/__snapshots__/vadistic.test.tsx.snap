// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`shorthand div 1`] = `
.emotion-0 {
  padding: 20px;
  background-color: silver;
}

<div
  className="emotion-0 emotion-1"
>
   Box 
</div>
`;
